src/js-marketing-notification.ts

```diff
@@ -89,11 +89,8 @@ export class JsMarketingNotification {
     const expansionClass = 'marketing-notification--expanded';
     node.classList.toggle(expansionClass);
 
-    if (node.classList.contains(expansionClass)) {
-      if (this.openCallback) this.openCallback(node);
-    }
-    else {
-      if (this.closeCallback) this.closeCallback(node);
-    }
+    if (node.classList.contains(expansionClass) && this.openCallback) return this.openCallback(node);
```

Comment:
@Izzmo Technically the callback properties could be null. They are set for our desired situation, but in the case of other implementations they wouldn't need to be set. So if I did not check for it there would be an error because there would be no error to call.